src/challenges.js

```diff
@@ -77,6 +156,38 @@ const limitFunctionCallCount = (cb, n) => {
 
 // Challenge 1: Go back to your prototype CuboidMaker and extend Cube using psuedo-classical inheritance to achiveve the same results you built using the ES6 class syntax
 
+function CuboidMaker(options) {
+  this.length = options.length;
+  this.width = options.width;
+  this.height = options.height;
+};
+
+CuboidMaker.prototype.volume = function volume() {
```

Comment:
Love the fact that you were able to comment out the classes and implement prototype.